src/pages/Login.jsx

```diff
@@ -1,3 +1,71 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import glamorous from 'glamorous';
 
-export default () => <div><h3>Login</h3></div>;
+const LOGINdiv = glamorous.div({
+    '& > .submit-button': {
+        background: '#337ab7',
+        color: '#fff',
+        padding: '5px 15px',
+        fontSize: 16,
+        '.disabled': {
+            background: '#888',
+            ':hover': {
+                cursor: 'none',
+            },
+        },
+    },
+});
+
+class LoginPage extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            username: '',
+            password: '',
+            submitted: false
+        };
+    }
+
+    render() {
+        const { username, password, submitted } = this.props;
+        return (
+            <LOGINdiv>
+                <div className={'login-field' + 
+                (submitted && !username ? 'has-error' : '')}>
```

Comment:
I like this.  We will want to add some in-line error reporting and this 'has-error' behavior will be really useful.